spec/models/group_lesson_summary_spec.rb

```diff
@@ -120,12 +120,12 @@
     before :each do
       subject = create :subject
       @group = create :group
-      @lesson = create(:lesson, group: @group, subject:)
+      @lesson = create(:lesson, group: @group, subject:, date: Time.zone.now)
       @first_student = create :student, group: @group
       @second_student = create :student, group: @group
       @ungraded_student = create :student, group: @group
       @deleted_student = create :student, group: @group, deleted_at: Time.zone.now
-      [@first_student, @second_student, @ungraded_student, @deleted_student].each { |s| create :enrollment, student: s, group: @group, active_since: 1.year.ago }
```

Comment:
This was unnecessary since students get an enrollment the moment they are created by the database trigger. What this does is give them another enrollment in the same group. Since StudentLessonSummary does an inner join with enrollments this caused it to double count the grades resulting in a failing test.